cumulusci/core/config/ScratchOrgConfig.py

```diff
@@ -224,10 +225,10 @@ def generate_password(self):
         p.run()
 
         stdout = []
-        for line in p.stdout:
+        for line in io.TextIOWrapper(p.stdout):
             stdout.append(line)
         stderr = []
-        for line in p.stderr:
+        for line in io.TextIOWrapper(p.stderr):
```

Comment:
can we shrink these to the comprehension form used in scratch_info?

Reply:
actually, is there a way we can generalize this collecting of sarge stdout&stderr into arrays of lines into a function or somethin?

Reply:
We can probably just do `stdout = io.TextIOWrapper(p.stdout).readlines()`

And sure, refactoring sounds good but let's do it separately